client/my-sites/themes/theme-filters.js

```diff
@@ -0,0 +1,203 @@
+/**
+* Functions for validating theme search filters.
+*/
+
+/**
+ * External dependencies
+ */
+import forIn from 'lodash/forIn';
+
+/* eslint-disable */
+/* Autogenerated object */
+const taxonomies = {
```

Comment:
If this is automatically generated then maybe it should be in its own file and imported like `import taxonomies from 'my-sites/themes/taxonomies';`. BTW I'm not sure if it should go in this directory or not since its technically a JSON object